.github/CONTRIBUTING.md

```diff
@@ -49,6 +67,9 @@ outlined on that page and do not file a public issue.
 * 4 spaces for indentation rather than tabs
 * Public classes and methods must contain header documentation
 * Use plain C functions whenever possible (as opposed to class methods)
+* Restrict subclassing (objc macro, final in examples)
```

Comment:
Hmm how & what I should put instead of the "actual macro"? And I will add the "Except when the class is designed for subclassing" but what is going to replace that sentence? @rnystrom